src/providers/ogr/qgsgeopackagedataitems.cpp

```diff
@@ -525,9 +525,23 @@ bool QgsGeoPackageAbstractLayerItem::deleteLayer()
   }
   else
   {
-    QMessageBox::information( nullptr, tr( "Delete Layer" ), tr( "Layer <b>%1</b> deleted successfully." ).arg( mName ) );
+
+    QgsGeoPackageConnectionItem *connectionParentItem = qobject_cast<QgsGeoPackageConnectionItem *>( mParent );
+    if ( connectionParentItem )
+    {
+      if ( QMessageBox::question( nullptr, QObject::tr( "Delete Layer" ), QObject::tr( "The layer <b>%1</b> has been deleted successfully."
```

Comment:
Oops, wrong choice. The correct grammar is "was deleted", not "has been" ;)

Reply:
Are you sure?
The decision was based on this:
https://github.com/qgis/QGIS/pull/8810#discussion_r245973658

Reply:
I have been / was quite sure about that as well. I guess @nyalldawson has to give a lesson about it next Hackfest ;-)

Reply:
Yep, I'm sure. I can't point to the rules explaining why sorry... But it just doesn't "sit right".

Reply:
Fair enough, @signedav can you prepare a PR?

Reply:
Yes.